src/app/dim-ui/Sheet.tsx

```diff
@@ -62,6 +62,11 @@ interface Props {
    * item popup don't close the popup they were spawned from!
    */
   allowClickThrough?: boolean;
+  /**
+   * Enables the ability click anywhere and drag the sheet down/up when the sheets contents are scrolled to the top.
+   * Defaults to true.
```

Comment:
Hmm, not sure I'm onboard. This means that the loadout sheet can't be dragged anymore?

Reply:
I can be dragged but only by the header.

Reply:
Before this, if you were scrolled to the top of the content (or the content didn't scroll) you could click **_anywhere_** on the loadout sheet and close it. When you are dragging items inside this resulted in some funky behaviour where the sheet would move slightly when you dragged an item.

Reply:
That's kind of what I suspected - and why I wasn't able to fix this before. That said, I'm a big fan of being able to drag the sheet to dismiss it. Is there any way to make this more targeted to just the items?

Reply:
I can have a go and see if I can tune the drop targets to be more mindful of other ones. Hopefully I will get a chance to look at it in the next couple of days.

I kind of don't like the grab sheet functionality as it is due to the conditional of being at the top of the sheet content but I imagine it is there for mobile touch scrolling? On desktop it is a bit awkward

Reply:
Yes, for mobile scrolling. You scroll up to the top then keep pulling to pull the sheet down.